library/testing/CMakeLists.txt

```diff
@@ -112,9 +112,5 @@ if(Qt5_FOUND)
   target_link_libraries(libf3dSDKTests Qt5::OpenGL)
 endif()
 
-if (UNIX AND NOT APPLE)
-  target_link_libraries(libf3dSDKTests stdc++fs)
```

Comment:
why not needed in the lib ?

Reply:
It's the testing executable, and we remove support of C++17 from it.

Reply:
right